examples/confirmation-mechanisms.js

```diff
@@ -0,0 +1,43 @@
+const { Client, CryptoTransferTransaction } = require("@hashgraph/sdk");
+
+async function main() {
+  const operatorPrivateKey = process.env.OPERATOR_KEY;
+  const operatorAccount = process.env.OPERATOR_ID;
+
+  if (operatorPrivateKey == null || operatorAccount == null) {
+    throw new Error(
+      "environment variables OPERATOR_KEY and OPERATOR_ID must be present"
+    );
+  }
+
+  const client = new Client({
+    network: { "0.testnet.hedera.com:50211": "0.0.3" },
+    operator: {
+      account: operatorAccount,
+      privateKey: operatorPrivateKey
+    }
+  });
+
+  // Confirmation 1 - Executing will run a "precheck"
+  const transactionId = await new CryptoTransferTransaction()
+    .addSender(operatorAccount, 0)
```

Comment:
Note that as of v1 you _can_ do this:

```js
  .addSender(operatorAccount, new Hbar(1))
  .addRecipient("0.0.1", new Hbar(1))
```

Just a plain number is still tinybar